recipes/jp-noui/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set version = "0.2.1" %}
+
+package:
+  name: jp-noui
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/j/jp_noui/jp_noui-{{ version }}.tar.gz
+  sha256: 4e1e8ba5c9aef02432559d234c751bddccfbf72ede3902ef7cd2dc9fcac50a1c
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
+
+
+requirements:
+  host:
+    - hatch-jupyter-builder
+    - hatch-nodejs-version
+    - jupyterlab >=3,<4
+    - nodejs
```

Comment:
if you are rebuilding after grabbing the sdist (not really recommended), at least specify an LTS nodejs version, e.g. `nodejs >=18,<19`

Reply:
I think it may only be used to get the version number. Trying to remove the nodejs dep by using hatch's pattern syntax

Reply:
> I think it may only be used to get the version number. Trying to remove the nodejs dep by using hatch's pattern syntax

yep that worked [4483b78](https://github.com/conda-forge/staged-recipes/pull/22083/commits/4483b780d464541ba6197470a14fa300033f98de)